src/sasctl/pzmm/write_json_files.py

```diff
@@ -181,22 +181,20 @@ def writeModelPropertiesJSON(
         eventProbVar : string, optional
             Model prediction metric for scoring. Default is None.
         jPath : string, optional
-            Location for the output JSON file. The default is the current
-            working directory.
+            Location for the output JSON file. The default is the current working directory.
         modeler : string, optional
-            The modeler name to be displayed in the model properties. The
-            default value is None.
+            The modeler name to be displayed in the model properties. The default value is None.
 
         Yields
-        ---------------
+        ------
```

Comment:
Same issue as with writeVarJSON() docstring, the Yields section is inappropriate here.

Reply:
Resolved.